cms/core/wagtail_hooks.py

```diff
@@ -20,4 +22,13 @@ def register_icons(icons: list[str]) -> list[str]:
     ]
 
 
+@hooks.register("insert_editor_js")
+def editor_js():
+    js_files = [
```

Comment:
note: let's merge this two into a `editor-tweaks.js` (or similarly named) file. Fewer things to load or look out for

Then this hook implementation becomes:

```python
@hooks.register("insert_editor_js")
def editor_js():
    return format_html('<script src="{}"></script>', static("js/editor-tweaks.js"))
```